README.md

```diff
@@ -32,26 +32,15 @@ cytosnake run <WORKFLOW>
 
 ## Installation
 
-First, install `CytoSnake` into your local machine:
+`CytoSnake` is can be installed by using `conda`.
 
 ```text
-git clone https://github.com/WayScience/CytoSnake.git
+conda install -c bioconda cytosnake
```

Comment:
maybe make the backtick type `bash` instead of `text`? not sure if this actually has any effect.